scripts/cmake/MCWAMP.cmake

```diff
@@ -37,12 +41,11 @@ endmacro(amp_target name )
 ####################
 # C++AMP runtime interface (mcwamp) 
 ####################
+pkg_check_modules(LIBUNWIND REQUIRED libunwind)
 macro(add_mcwamp_library name )
-  set(CMAKE_CXX_COMPILER "${PROJECT_BINARY_DIR}/compiler/bin/clang++")
-  add_compile_options(-std=c++11)
   add_library( ${name} ${ARGN} )
   amp_target(${name})
-  target_link_libraries(${name} unwind)
+  target_link_libraries(${name} ${LIBUNWIND_LIBRARIES})
```

Comment:
How would that be used in hcc-config.cmake. MCWAMP.cmake is not included in hcc-config.cmake, it might not even be present on the system building downstream libraries.